src/server/utils/wallets/getGcpKmsAccount.ts

```diff
@@ -40,14 +40,28 @@ export async function getGcpKmsAccount(
 ): Promise<GcpKmsAccount> {
   const { name: unprocessedName, clientOptions, client } = options;
 
+  let rawPrivateKey: string | undefined;
+  if (clientOptions?.credentials) {
+    if ("private_key" in clientOptions.credentials) {
+      rawPrivateKey = clientOptions.credentials.private_key;
+    }
+  }
```

Comment:
IMO a bit clearer to just "sanitize" the private key in clientOptions and continue passing the clientOptions below.

```typescript
  if (clientOptions?.credentials) {
    if ("private_key" in clientOptions.credentials) {
      clientOptions.credentials.private_key =
        clientOptions.credentials.private_key?.split(String.raw`\n`).join("\n");
    }
  }
```